base-template/about-base-template.md

```diff
@@ -11,84 +24,349 @@ The base template goes one step further, in that it helps you create any templat
+
+}
+
+## Customization
+
+{Template author tip:
+
+* There are often multiple approaches to consider when implementing a doctype.
+* In this section you should discuss different approaches that can be applied to this doctype, including logic and recommendations to help a docset owner select the best strategy for them.
+* Often there would be a light version of this template to use for immature projects, and a more comprehensive version for mature projects. You should discuss how a project might mature their docs from light to comprehensive versions.
+
+_{TBD: Add further guidance and sample text here.}_
+
+}
+
+{Docset owner tip:
+
+* There are multiple approaches which can be taken to customize this template. These may depend upon:
+    * The maturity of your project.
+    * The size, interest, and skillsets of your contributor community.
+    * The documentation goals you have for the project.
+* This section discusses approaches you may take to customize this template to your project.
+
+}
+
+{Template author tip:
+
+* Add content here. …}
+
+}
+
+## Maintenance strategy
+
+The long term health of a project’s documentation depends upon the project's ability to establish sustainable maintenance processes for each doctype.
+
+* If content on a site is inaccurate or dated, it will reduce the trustworthiness of all the content on the site, as the user typically won’t be able to determine which of the content is trustworthy.
+* As such, a project should only create content where there is commitment for continued maintenance of the content.
+
+This section suggests maintenance strategies which can be applied for {doctype} documents.
+
+{Docset owner tip:
+*  You should select and set up an appropriate strategy for maintaining {doctype} documents. This strategy should be described in your Documentation Management Plan.
 
-- GitHub to download, edit, and use the base template.
-- Markdown to create and edit the base template to suit your requirements.
-- Different documentation types to identify if you need to create a new template or edit an existing one.
-- Technical writing to produce a well-written document that includes the relevant reference aspects.
+}
 
-## Who will need the base template
+_{TBD: We should define a template for a Documentation Management Plan, then reference from here.}_
 
-The base template is for:
+{Template author tip:
 
-- Internal users - doc type template authors who wish to align and/or contribute to The Good Docs Project (GDP).
-- External users - the community outside of the GDP.
+* In this section, you should recommend strategies for maintaining {doctype} documentation. This could include:
+* Automated documentation generation from code.
+* Automated reports which highlight:
+    * When documentation is out of date with software.
+    * When documentation hasn’t been reviewed for a while.
+* Integrating documentation checklists into software deployment, build, and test processes.
```

Comment:
@Loquacity , I think these are strategies which are applied at a "Whole Docset" level. As such they would fit within a "Documentation Management Plan" or maybe within the "Information Architecture Guide".
If you think it fits here, feel free to suggest words to add.
I'll close out this comment for the moment (mainly because I can't think of how to address this in this doc at the moment). We can add into a future iteration.